### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by smriti-porwal**
Built Team App
1. Added header with Team name and logo.
2. Added card component, each card having detail of team members.

### CODE REVIEW

File: Assignments/src/Components/CardComponent/card-component.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import './card-component.css';
+
+const CardComponent = (props) => {
```
Comment (nitishnivedan) [inline Assignments/src/Components/CardComponent/card-component.js:3]: can de structure to one more level. in function parameter. you can write like this:
const CardComponent = ({data})=>{
......
}

File: Assignments/src/Components/BodyComponent/body-component.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import CardComponent from "../CardComponent/card-component";
+import "./body-component.css";
+
+const BodyComponent = () => {
+  let teamMemberDetails = [
```
Comment (nitishnivedan) [inline Assignments/src/Components/BodyComponent/body-component.js:5]: should use const. if the values are not changing.

**Comment by shaksham08** [comment]
Looks good to me

**Review by shaksham08 — APPROVED** [review]

File: Assignments/src/Components/CardComponent/card-component.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import './card-component.css';
+
+const CardComponent = ({data}) => {
+    const {photo, name, place, company, designation} = data;
+    return (
+        <div className="card">
+            <img src={photo} className="userimage" alt="UserName"></img>
```
Comment (ap221882) [inline Assignments/src/Components/CardComponent/card-component.js:7]: img is a self closing html tag.

**Review by ap221882 — COMMENTED** [review]
Well written with nice comments. Kudos!

**Pull request merged**